isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -83,22 +83,22 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
       <li>Finally, setup the environment variables:
```

Comment:
The updates to this file make sense, but I don't think there's any way to access this page via https://isis.astrogeology.usgs.gov/Installation/index.html any more and I don't think it's being kept up to date.

Reply:
Agreed, we should consider removing a lot of the documentation I think